### DISCUSSION TITLE
Slow subscribers getting disconnected.

### DISCUSSION THREAD

**Discussion by huguesBouvier** [emqx/emqx#10374] [Q&A]
config: EMQX v5, 5 nodes and 5 cores nodes:
![image](https://user-images.githubusercontent.com/33438817/231243239-8a596e25-ec63-4304-a7d6-a24b4ba6ea31.png)

my config is:
apiVersion: apps.emqx.io/v2alpha1
kind: EMQX
metadata:
   name: emqx
spec:
   image: emqx:5.0
   listenersServiceTemplate:
    spec:
      type: LoadBalancer
   coreTemplate:
     spec:
       replicas: 5
   replicantTemplate:
     spec:
       replicas: 0

I run a simple test:
./emqtt_bench sub -c 1 -h  10.0.0.15 -t "#"   -q 1
./emqtt_bench pub -c 5 -h  10.0.0.15 -t  bench/%i -I 0 -s 255000 -q 1  (Means 5 clients of 255M).

I see that 4 clients get disconnected:
![image](https://user-images.githubusercontent.com/33438817/231243677-ba8d3302-bb8f-4058-b66e-ca7c43480a7f.png)

From my observations, it happens because the subscriber can't keep up with the publishers. But it is pretty annoying. I there a way to make it work without me having to manually throttle my publishers?

**Comment by ieQu1** [comment]
Hello,

~I don't think this is an EMQX issue: `{shutdown, tcp_closed}` error means the remote side (the load-balancer) closed the connection. Perhaps one solution is to increase idle timeout on the load balancer. If you suspect that the subscriber cannot match the throughput of the publishers, it makes sense to use shared subscription.~

Please disregard the above message. I overlooked the fact that `{shutdown, tcp_closed}` was reported on the client side rather than broker side.

Can you share logs from the broker side?
